sycl/doc/extensions/deprecated/sycl_ext_intel_online_compiler.asciidoc

```diff
@@ -0,0 +1,203 @@
+= SYCL Intel extension: Online Compilation
+Konstantin Bobrovskii <konstantin.s.bobrovsky@intel.com>, John Pennycook <john.pennycook@intel.com>
+v0.1
+:source-highlighter: pygments
+:icons: font
+== Introduction
+This document describes an interface for online compilation from high-level languages, such as
+OpenCL, to a binary format, such as SPIR-V, loadable by SYCL backends. Unlike SYCL 2020 provisional's
+OpenCL backend online compilation interface, this interface is not bound to any particular backend and does
+not require available SYCL context for online compilation.
+
+This gives a flexibility to "cross-compile" to SPIR-V or other supported formats without any SYCL
+device or context available. The online compilation API uses the `online_compiler` class to access
+compilation services. Instances of the class are constructed based on a specification of the desired
+compilation target passed to the constructors - such as compiled code format, target architecture,
+etc. All the settings are optional, and by default the target is generic SPIR-V.
+
+This API is an Intel SYCL extension.
+
+== Status
```

Comment:
20-22 is newly added here.
Unfortunately since I need to add a few lines about deprecation to the original file git do not track this as movement of file with update.
other lines are the same